src/tests/meetings.test.js

```diff
@@ -0,0 +1,194 @@
+            .post(MEETING_API_URL)
+            .send({
+                match_id: null,
+                meeting_date: '2016-02-05T03:30:17.883Z',
+                location: 'New York',
+                notes: 'some meeting',
+                rating: Math.floor((Math.random()*5))
+            });
+            expect(res.status).toEqual(400);
+        }
+
+        it('should return status 400 on fail (no meeting date)'), async () => {
+            const res = await request(server)
+            .post(MEETING_API_URL)
+            .send({
+                match_id: 1,
+                meeting_date: '',
+                location: 'New York',
+                notes: 'some meeting',
+                rating: Math.floor((Math.random()*5))
+            });
+            expect(res.status).toEqual(400);
+        }
+
+        it('should return status 400 on fail (no location)'), async () => {
+            const res = await request(server)
+            .post(MEETING_API_URL)
+            .send({
+                match_id: 1,
+                meeting_date: '2016-02-05T03:30:17.883Z',
+                location: '',
+                notes: 'some meeting',
+                rating: Math.floor((Math.random()*5))
+            });
+            expect(res.status).toEqual(400);
+        }
+
+
+        it('should return error on fail (no match id)'), async () => {
+            const res = await request(server)
+            .post(MEETING_API_URL)
+            .send({
+                match_id: null,
+                meeting_date: '2016-02-05T03:30:17.883Z',
+                location: 'New York',
+                notes: 'some meeting',
+                rating: Math.floor((Math.random()*5))
+            });
+            expect(res.body).toEqual({error: 'Please fill out the required fields'});
+        }
+        
+        it('should return error on fail (no meeting date)'), async () => {
+            const res = await request(server)
+            .post(MEETING_API_URL)
+            .send({
+                match_id: 1,
+                meeting_date: '',
+                location: 'New York',
+                notes: 'some meeting',
+                rating: Math.floor((Math.random()*5))
+            });
+            expect(res.body).toEqual({error: 'Please fill out the required fields'});
+        }
+
+        it('should return error on fail (no location)'), async () => {
+            const res = await request(server)
+            .post(MEETING_API_URL)
+            .send({
+                match_id: 1,
+                meeting_date: '2016-02-05T03:30:17.883Z',
+                location: '',
+                notes: 'some meeting',
+                rating: Math.floor((Math.random()*5))
+            });
+            expect(res.body).toEqual({error: 'Please fill out the required fields'});
+        }
+        
+    })
+
+    describe('GET ROUTE /MEETINGS/:id', () => {
+        it('should return status 200 on success', async () => {
```

Comment:
as with each test - make sure you create test seeds at the beginning of your test because the table is actually empty